docs/source/a_quick_tour.mdx

```diff
@@ -200,6 +199,47 @@ The `combine` function accepts both the list of names of the metrics as well as
 }
 ```
 
+## Running evaluation on a suite of tasks
+
+It can be useful to evaluate models on a variety of different tasks to understand their downstream performance. The [EvaluationSuite](evaluation_suite) enables evaluation of models on a collection of tasks. Tasks can be constructed as ([evaluator](base_evaluator), dataset, metric) tuples and passed to an [EvaluationSuite](evaluation_suite) stored on the Hugging Face Hub as a Space. See the [evaluator documentation](base_evaluator) for a list of currently supported tasks.
+
+`EvaluationSuite` scripts can be defined as follows, and supports Python code for data preprocessing.
+
+```python
+import evaluate
+from evaluate.evaluation_suite import SubTask
+
+class Suite(evaluate.EvaluationSuite):
+
+    def __init__(self, name):
+        super().__init__(name)
+        self.preprocessor = lambda x: {"text": x["text"].lower()}
+        self.suite = [
+            SubTask(
+                task_type="text-classification",
+                data="glue",
+                subset="cola",
+                split="test[:10]",
+                args_for_task={
+                    "metric": "accuracy",
+                    "input_column": "sentence",
+                    "label_column": "label",
+                    "label_mapping": {
+                        "LABEL_0": 0.0,
+                        "LABEL_1": 1.0
+                    }
+                }
+            )]
+```
+
+Evaluation can be run by loading the `EvaluationSuite` and calling `.suite.run()` with a model or pipeline.
+
+```python
+from evaluate import EvaluationSuite
+suite = EvaluationSuite.load('mathemakitten/glue-evaluation-suite')
+results = suite.run("gpt2")
```

Comment:
I think it would be nice to show that the result looks like.

Reply:
Added!